04-account-kit-manage-permissions/index.ts

```diff
@@ -0,0 +1,66 @@
+// SDK exposes all you need
+import { Chains, PublicKey, Permission } from '@[SOMETHING]kit'
+import { Account } from '@[SOMETHING]kit/account'
+import { Session } from '@[SOMETHING]kit/session'
+
+async function main() {
+    const session = Session.login();
+    const account: Account = await session.getAccount();
+
+    // or
+    // const account: Account = await Account.load("teamgreymass", Chains.EOS)
+
+    // Manage account Permissions
+
+    // Get a permission
+    const permission: Permission = account.getPermission("active")
```

Comment:
As soon as you want to have dot notation to access properties that can be lazy loaded we run in the the problem we discussed at length of handling loading for the developers vs punting it to them and requiring guards as soon as you want to interact with an account object since data may or may not be present.